src/reference/asciidoc/whats-new.adoc

```diff
@@ -66,3 +66,9 @@ See <<file-writing-file-names>> for more information.
 The outbound endpoints now support a `RabbitTemplate` configured with a `ContentTypeDelegatingMessageConverter` such
 that the converter can be chosen based on the message content type.
 See <<content-type-conversion-outbound>> for more information.
+
+==== Redis Changes
+
+Previously, the queue channel adapters always used the Redis List in a fixed direction, pushing to the left end and reading from the right end.
+It is now possible to configure the reading and writing direction.
```

Comment:
... direction using `rightPop` and `leftPush` options respectively.

It is good mention them here. For example, I read only `What's New` for libraries which I use . Everything else I can get from their JavaDocs and source code. :smile: